webcompat/templates/nav/addon-links.html

```diff
@@ -41,9 +32,10 @@
     </li>
     {%- else -%}
     <li class="nav-item">
-        <a class="nav-link" href="https://github.com/webcompat/webcompat.com/issues?state=open" title="Navigate to GitHub issues">
+        <a class="nav-link" href="https://github.com/webcompat/webcompat.com/issues?state=open"
+            title="Navigate to GitHub issues">
             <span class="link-text">Give Feedback</span>
         </a>
     </li>
     {%- endif -%}
-</ul>
+</ul>
```

Comment:
(i guess my vscode did some auto-formatting, let me know if you want me to revert these unrelated style changes)

Reply:
You are a terrible person. 👹

Reply:
(i mean, we already knew that...)